Sources/Public/Pretty.swift

```diff
@@ -148,9 +147,9 @@ extension Pretty {
 
     // MARK: - private
 
-    private static func output(message: String) {
+    private static func output(message: String, option: Option = sharedOption) {
```

Comment:
I think that `option` are should avoid to use default arguments (`= sharedOption`) because can be missed to pass to arguments when call to this function.

Actually, missed to pass to arguments at L35 in this source.

Reply:
fixed it.